virttest/libvirt_xml/vm_xml.py

```diff
@@ -802,6 +802,26 @@ def get_disk_attr(vm_name, target, tag, attr, virsh_instance=base.virsh):
 
         return attr_value
 
+    def set_disk_attr(self, vm_name, target, tag, attr):
+        """
+        Set value of disk tag attributes for a given target dev.
+
+        :param vm_name: domain name
+        :param target: dev of the disk
+        :param tag: disk tag
+        :param attr: the tag attribute dict to set
+        """
+        key = ""
+        try:
+            disk = self.get_disk_all()[target]
+            if tag in ["driver", "boot", "address", "alias", "source"]:
```

Comment:
There are many disk attributes , you can check ```__slots__``` in libvirt_xml/devices/disk.py
And I prefer to add these kind of functions to utils_test/libvirt.py as the xml related modules here already a bit of mess.